app/views/content/train-to-be-a-teacher/_why-teach.html.erb

```diff
@@ -1,10 +1,11 @@
 <section class="why-teach" aria-label="Why teach?">
   <div class="why-teach__container">
     <div class="why-teach__content">
-      <h2 class="heading-l">
-        <div class="caption-l caption--bold caption--purple">Why teach?</div>
-        Inspire the next generation
-      </h2>
+      <%= render Content::HeadingComponent.new(
+        heading: "Inspire the next generation",
+        tag: :h2,
+        caption: "Why teach?",
```

Comment:
@gazcarr This caption (Why teach?) is quite a bit smaller now - are you happy with this? I'm fine either way

https://review-get-into-teaching-app-2975.london.cloudapps.digital/train-to-be-a-teacher